22nd-jan/cacheFunction.js

```diff
@@ -0,0 +1,24 @@
+Utils = {
+  memoize: function (param) {
+    if (param in this.obj.cache) {
```

Comment:
Using `in` is not a good idea. Same reason why for in could be mis-used.
https://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-a-bad-idea